pages/dashboard/follows.vue

```diff
@@ -64,11 +64,12 @@ export default {
       data.$auth.headers
     )
 
-    const mods = (
+    let mods = (
```

Comment:
Why not simply chain the calls instead of mutating the variable?

Reply:
It just made more sense conceptually to me, but I can chain the calls.